src/include/miopen/batched_transpose_sol.hpp

```diff
@@ -28,11 +28,27 @@
 
 #include <miopen/miopen.h>
 #include <miopen/errors.hpp>
+#include <miopen/invoke_params.hpp>
 #include <miopen/kernel_info.hpp>
 #include <miopen/op_kernel_args.hpp>
 #include <miopen/execution_context.hpp>
 #include <vector>
 
+struct transpose_invoke_param : public miopen::InvokeParams
+{
+    ConstData_t src = nullptr;
+    Data_t dst      = nullptr;
+
+    transpose_invoke_param(ConstData_t src_, Data_t dst_) : src(src_), dst(dst_) {}
```

Comment:
This constructor doesn't call the base constructor explicitly. Are we calling the `no-arg` constructor here? Better to make it explicit.

Reply:
Not my code. InvokeParams is a very basic wrapper struct defining no explicit ctors and containing only an InvokeType which defaults to Run. I'm not familiar with the language vs. compiler-specific rules behavior regarding auto-generated ctors, but it does work, so OK.

Reply:
You should always call some base class constructor in a derived constructor as that's a good practice and structs and classes are exactly the same in C++ except for default visibility.

Reply:
It is called by compiler implicitly - the default one.